lib/ansible/modules/system/mount.py

```diff
@@ -57,18 +57,22 @@
     default: 0
   state:
     description:
-      - If C(mounted), the device will be actively mounted and appropriately
-        configured in I(fstab). If the mount point is not present, the mount
+      - If C(mount), the device will be actively mounted without changing
+        I(fstab_file). If the mount point is not present, the mount
         point will be created.
-      - If C(unmounted), the device will be unmounted without changing I(fstab).
+      - If C(mounted), the device will be actively mounted and appropriately
+        configured in I(fstab_file). If the mount point is not present, the
+        mount point will be created.
+      - If C(unmounted), the device will be unmounted without changing
+        I(fstab_file).
       - C(present) only specifies that the device is to be configured in
-        I(fstab) and does not trigger or require a mount.
+        I(fstab_file) and does not trigger or require a mount.
       - C(absent) specifies that the device mount's entry will be removed from
-        I(fstab) and will also unmount the device and remove the mount
+        I(fstab_file) and will also unmount the device and remove the mount
         point.
     required: true
-    choices: [ absent, mounted, present, unmounted ]
-  fstab:
+    choices: [ absent, mounted, present, mount, unmounted ]
+  fstab_file:
```

Comment:
I don't see a reason to rename this option to `fstab_file`. Please change it back to `fstab`.